packages/shared/sdk-server/src/data_sources/defaultHeaders.ts

```diff
@@ -12,7 +12,9 @@ export default function defaultHeaders(
   const sdkData = info.sdkData();
   const headers: { [key: string]: string } = {
     authorization: sdkKey,
-    'user-agent': `NodeJSClient/${sdkData.version}`,
+    'user-agent': `${sdkData.userAgentBase ? sdkData.userAgentBase : 'NodeJSClient'}/${
+      sdkData.version
+    }`,
```

Comment:
Eslint complained about using unnecessary ternary here, so to fix that:

```suggestion
'user-agent': `${userAgentBase ?? 'NodeJSClient'}/${version}`,
```